pages/index.tsx

```diff
@@ -0,0 +1,560 @@
+import { FunctionComponent } from 'react';
+
+import { H1, H3 } from '../styles/MDComponents';
+import PageLink, { CallToAction } from '../components/PageLink';
+
+import AnalyticsManagerIcon from '../images/svg/analyticsmanager.svg';
+import DevIcon from '../images/svg/dev.svg';
+
+import styles from './index.module.scss';
+
+function emptyRowElement() {
+  return <div style={{ display: 'flex', padding: '15px' }} />;
+}
+
+const LandingPage: FunctionComponent = () => {
+  return (
+    <div style={{ width: '85%' }}>
+      <H1>Getting started with Avo</H1>
+      <p className={styles.text}>
+        We distinguish 3 steps to adopt Avo. You don't need to go through all
+        the steps to receive value, because every step has value in itself.
+      </p>
+
+      <div className={styles.tocBg}>
+        <p>
+          <a
+            href="#stage1"
+            className={[styles.tocItem, styles.avoPink].join(' ')}
+          >
+            1. Better tracking plan management today - no code solution
```

Comment:
I'm thinking something along the lines of: 

1. Tracking plan management workflows – no engineering required
2. Tracking plan management workflows with validation – one off SDK install required
3. End-to-end tracking plan collaboration and implementation workflows – auto generated analytics code used to implement

Maybe too long? What do you think @alex-tpom6oh?

Reply:
What do you think about this @thoragudf 

          1. Tracking plan management – no engineering required
          2. Tracking plan management with validation – one time SDK install required
          3. End-to-end tracking plan collaboration and implementation workflows – requires engineers to adopt Avo into their workflow

It feels like 3rd requirement can be improved 🤔

Reply:
What about something like:

1. Tracking plan management – no engineering required
2. Tracking plan validation – one time SDK install
3. Tracking plan release workflow for your team – team wide Avo adoption

Reply:
Maybe just `3. Tracking plan implementation workflow for your team`?